templates/mcq.jsx

```diff
@@ -0,0 +1,104 @@
+import Adapt from 'core/js/adapt';
+import { templates, classes, html, compile } from 'core/js/reactHelpers';
+
+export default function(model, view) {
+  const data = model.toJSON();
+  const ariaLabels = Adapt.course.get('_globals')._accessibility._ariaLabels;
+
+  const isInteractive = model.isInteractive();
+
+  return (
+    <div className='component__inner mcq__inner'>
+
+      {templates.component(model, view)}
+
+      <div
+        className={classes([
+          'component__widget',
+          'mcq__widget',
+          !data._isEnabled && 'is-disabled',
+          data._isInteractionComplete && 'is-complete is-submitted show-user-answer',
+          data._isCorrect && 'is-correct'
+        ])}
+        role={data._isRadio ? 'radiogroup' : 'group'}
+      >
+
+        {data._items.map(({ text, _index, _isActive, _shouldBeSelected, _isHighlighted }, index) =>
+
+          <div
+            className={classes([
+              `mcq__item item-${index}`,
+              (!isInteractive && data._canShowMarking && _shouldBeSelected) ? 'is-correct' : null,
+              (!isInteractive && data._canShowMarking && !_shouldBeSelected) ? 'is-incorrect' : null
+            ])}
+            key={_index}
+          >
+
+            <input
+              className='mcq__item-input'
+              id={`${data._id}-${index}-input`}
+              name={data._isRadio ? `${data._id}-item` : null}
+              type={data._isRadio ? 'radio' : 'checkbox'}
+              disabled={!data._isEnabled}
+              aria-label={(isInteractive || !data._canShowMarking) ?
+                Adapt.a11y.normalize(text) :
+                `${_shouldBeSelected ? ariaLabels.correct : ariaLabels.incorrect}, ${_isActive ? ariaLabels.selectedAnswer : ariaLabels.unselectedAnswer}. ${Adapt.a11y.normalize(text)}`}
+              data-adapt-index={_index}
+              onKeyPress={(event) => view.onKeyPress(event)}
```

Comment:
Losing the brackets might make it slightly cleaner to read?
```suggestion
              onKeyPress={event => view.onKeyPress(event)}
```

Reply:
Maybe? I've been putting brackets around a single parameter lately as a matter of course since I realised that when I left them out in development then went back to add another param, I'd forget to add brackets and spend time working out an avoidable error. That's a personal thing though so am happy to go with consensus.

Reply:
Happy to go with your pers pref!